content/en/index.md

```diff
@@ -14,11 +14,12 @@ It’s free to use for all developers and software providers, with Sigstore’s
 
 ## How to use Sigstore
 
+* Install Cosign - [Cosign Installation page](/cosign/installation/)
```

Comment:
Good with me. Were we going to link to Verify still?

Reply:
I put in a line on index.md that says "Once installed, Cosign can also be used to verify. If you need to verify a signed artifact, install Cosign and follow the verify instructions."

Reply:
I think maybe all these bullet points should go away.   Unless we want to tell people to use the policy controller?   If we want something that says:

I Want to sign something.
I Want to verify something.

with links to the proper places?   Would that be good?

Reply:
I think that's super clear. Would everyone be okay with limiting the "set off on the correct path" options to those two main user journeys?

Reply:
I changed it a little from the two simple points because you wanting to be able to sign blobs and sign containers are two different things..   I also left in the gitsign thing and putin the I want to verify something.  So the bullet list looks like this now:


I want to [install and Quick Start](https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/cosign/overview/#quick-start)
I want to [sign a blob](https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/content/en/cosign/signing_with_blobs)
I want to [sign a container](https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/content/en/cosign/signing_with_containers)
I want to [Sign Git commits with Gitsign](https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/gitsign/overview) - [Quick Start](https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/gitsign/overview/#quick-start)
I want to [verify entries with Cosign](https://github.com/jonvnadelberg/sigstore-docs/blob/update-index.md/cosign/verify)


Let me know if more changes are wanted, or we want to go to back to a simpler thing.